common/junit-platform-native/src/main/resources/initialize-at-buildtime

```diff
@@ -0,0 +1,119 @@
+java.beans.Introspector
```

Comment:
There is
```
rci.initializeAtBuildTime("java.beans.Introspector", "JMX support");
```
in `substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/jdk/JmxCommonFeature.java`
Instead of adding `java.beans.Introspector` here, we should unconditionally set it to initializeAtBuildTime in the image builder (in `com.oracle.svm.hosted.jdk.JDKInitializationFeature#afterRegistration`).

Reply:
cc @christianhaeubl